salt/grains/core.py

```diff
@@ -1260,11 +1260,16 @@ def os_data():
     # pylint: disable=unpacking-non-sequence
     (grains['kernel'], grains['nodename'],
      grains['kernelrelease'], version, grains['cpuarch'], _) = platform.uname()
+
+    # pylint: enable=unpacking-non-sequence
+    grains['kernelversion'] = platform.version()
```

Comment:
Is there a difference between `version` (which is the fourth element in the tuple returned from `platform.uname()`) and `platform.version()`? If they are the same, just use `version` or plug `grains['kernelversion']` right into the output of `platform.uname()` instead of `version`. It seems like @wolfpackmars2 has done something exactly like this in #40782.